packages/eslint-config-eslint/default.yml

```diff
@@ -262,6 +266,17 @@ rules:
     semi: "error"
     semi-spacing: ["error", {before: false, after: true}]
     semi-style: "error"
+
+    sonarjs/max-switch-cases: "off"
+    sonarjs/no-collapsible-if: "off"
+    sonarjs/no-duplicate-string: "off"
+    sonarjs/no-identical-functions: "off"
+    sonarjs/no-small-switch: "off"
+    sonarjs/prefer-immediate-return: "off"
+    sonarjs/cognitive-complexity: "off"
+    sonarjs/no-nested-switch: "off"
+    sonarjs/no-nested-template-literals: "off"
```

Comment:
Can we keep these lists alphabetically sorted? There's a VSCode extension that enables a right-click to sort lines option.

Reply:
I've sorted them, though I moved the plugin rules separate from the regular rules to allow easier separate consideration.